### PR TITLE
GH-597: An empty JSON response should be `[]` instead of ``.

### PR DESCRIPTION
**Pull request by tbl3rd**
https://broadinstitute.atlassian.net/browse/GH-597

### CODE REVIEW

**Review by pshapiro4broad — DISMISSED** [review]
A regression test would be nice, if it's not too difficult to add. Given the number of tests clio already has I was surprised that we didn't already have a test that covered this case.

**Comment by tbl3rd** [comment]
I meant to open this as a draft. Sorry.

File: clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala [hunk 1]
```diff
@@ -64,26 +65,39 @@ object ClioClient extends LazyLogging {
...
+    }
+
+    def complete(done: Try[Done]): Unit = {
+      val status = done match {
+        case Success(_) => 0
+        case Failure(ex) =>
+          logger.error("Failed to execute command", ex)
+          1
+      }
+      Await.result(system.terminate(), 23.seconds)
```
Comment (pshapiro4broad) [inline clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala:91]: It looks like this is the only change here, is that correct?

Maybe a comment would be helpful, at least for the magic number.

**Review by pshapiro4broad — DISMISSED** [review]
Looks good, as far as I can tell.

File: clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala (hunk 1 above)
Comment (tbl3rd) [reply clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala:91]: I thought the magic number was magic enough for its value to be obviously arbitrary, but OK.

**Review by rhiananthony — APPROVED** [review]
👍

**Pull request merged**